src/contrib/mlir/core/pass/ng_dialect_builder.cpp

```diff
@@ -264,6 +264,7 @@ mlir::Type NgDialectConversionPass::getMlirType(const element::Type& type)
     {
     case ngraph::element::Type_t::undefined:
     case ngraph::element::Type_t::dynamic:
+    case ngraph::element::Type_t::u1:
```

Comment:
You could map it to u8 here (check out boolean below).

Reply:
size of `u1` is 1 that is differ from size of u8. Also it's not related to my initial purpose (just to add u1 precision), so let's handle it in future.

Reply:
size of u1 is 1 that is differ from size of u8. Also it's not related to my initial purpose (just to add u1 precision), so let's handle it in future.